packages/button/package.json

```diff
@@ -19,7 +19,10 @@
     "test:watch": "npm run test -- --watchAll"
   },
   "jest": {
-    "setupTestFrameworkScriptFile": "<rootDir>/test/test-setup.js"
+    "setupFiles": [
```

Comment:
Kinda late for this question, I suppose.  Do we like jest setup in package.json or should we use jest.config.js files?

Reply:
move config to a separate js file